armi/physics/neutronics/globalFlux/globalFluxInterface.py

```diff
@@ -102,23 +102,30 @@ def interactEveryNode(self, cycle, node):
         is up to date with the reactor state.
         """
         interfaces.Interface.interactEveryNode(self, cycle, node)
-
-        if self.r.p.timeNode == 0:
-            self._bocKeff = self.r.core.p.keff  # track boc keff for rxSwing param.
+        self._setRxSwingRelatedParams()
 
     def interactCoupled(self, iteration):
         """Runs during a tightly-coupled physics iteration to updated the flux and power."""
         interfaces.Interface.interactCoupled(self, iteration)
+        self._setRxSwingRelatedParams()
+
+    def _setRxSwingRelatedParams(self):
+        """Set Params Related to Rx Swing"""
         if self.r.p.timeNode == 0:
-            self._bocKeff = self.r.core.p.keff  # track boc keff for rxSwing param.
-
-    def interactEOC(self, cycle=None):
-        interfaces.Interface.interactEOC(self, cycle)
-        if self._bocKeff is not None:
-            self.r.core.p.rxSwing = (
-                (self.r.core.p.keff - self._bocKeff)
-                / self._bocKeff
-                * units.ABS_REACTIVITY_TO_PCM
+            # track boc uncontrolled keff for rxSwing param.
+            self._bocKeff = self.r.core.p.keffUnc or self.r.core.p.keff
+
+        # A 1 burnstep cycle would have 2 nodes, and the last node would be node index 1 (first is zero)
+        lastNodeInCycle = getBurnSteps(self.cs)[self.r.p.cycle]
+        if self.r.p.timeNode == lastNodeInCycle and self._bocKeff is not None:
+
+            eocKeff = self.r.core.p.keffUnc or self.r.core.p.keff
+            swing = (eocKeff - self._bocKeff) / (eocKeff * self._bocKeff)
+            self.r.core.p.rxSwing = swing * units.ABS_REACTIVITY_TO_PCM
+            runLog.important(
```

Comment:
Does this need to be `important` or could it be something like `extra` that you toggle the verbosity to when needed? (I don't have a sense of what analysts want to see)

Reply:
I agree. I'm not thrilled that every PR people add more and more log statements.

But `important` is usually reserved for things that may-or-may-not be bugs, and people better pay attention to.  If we are keeping this log statement, I would say `extra`.

Reply:
info is OK. its defiantly more important than `extra`.

Reply:
offline, decided to go with info!